src/include/OpenImageIO/simd.h

```diff
@@ -2347,6 +2370,61 @@ OIIO_FORCEINLINE float4 max (const float4& a, const float4& b)
 
 
 
+/// andnot(a,b) returns ((~a) & b)
+OIIO_FORCEINLINE float4 andnot (const float4& a, const float4& b) {
+#if defined(OIIO_SIMD_SSE)
+    return _mm_andnot_ps (a.simd(), b.simd());
+#else
+    const int *ai = (const int *)&a;
+    const int *bi = (const int *)&b;
+    return bitcast_to_float4 (int4(~(ai[0]) & bi[0],
+                                   ~(ai[1]) & bi[1],
+                                   ~(ai[2]) & bi[2],
+                                   ~(ai[3]) & bi[3]));
+#endif
+}
+
+
+
+/// Fused multiply and add: (a*b + c)
+OIIO_FORCEINLINE float4 madd (const simd::float4& a, const simd::float4& b,
+                              const simd::float4& c)
+{
+#if OIIO_SIMD_SSE && __FMA__
```

Comment:
What about creating a special `OIIO_FMA_ENABLED` macro that digests the compiler specific `__FMA__` into something portable? Just thinking about MSVC ...